docs/rfcs/XeTile.md

```diff
@@ -208,6 +207,22 @@ With the data being presented as 4D vector, all the vector based XeTile operatio
 ```
 The tile_pack and tile_unpack operation is similar to pack and unpack operation of tensor dialect. The source vector must be a 2D dimension vector, and no permutation is allowed for the result 4D vector, so effectively the blocking effect is identical to tensor pack/unpack operation with inner_dims_pos = [0,1] inner_dims_pos = [0, 1].
 
+## support for load_gather and store_scatter (experimental)
+`load_gather` (aka. load) loads data with each element's address being explictly specified. The tile is created with a base address and offset for each element to be load. The result tile has a `scatter` attribute to different it from the regular tile.
+```mlir
+  %tile0 = XeTile.init_tile %base_addr, %tile_offsets:
```

Comment:
XeTile->xetile here and below since it is a name for dialect operation

Reply:
fixed.